doc/src/manual/arrays.md

```diff
@@ -410,25 +410,32 @@ julia> searchsorted(a, 4)
 
 ## Assignment
 
-The general syntax for assigning values in an n-dimensional array A is:
+The general syntax for assigning values in an n-dimensional array `A` is:
 
 ```
-A[I_1, I_2, ..., I_n] = X
+A[I₁, I₂, ..., Iₙ] = X
 ```
 
-where each `I_k` may be a scalar integer, an array of integers, or any other
+where each `Iₖ` may be a scalar integer, an array of integers, or any other
 [supported index](@ref man-supported-index-types). This includes
 [`Colon`](@ref) (`:`) to select all indices within the entire dimension,
 ranges of the form `a:c` or `a:b:c` to select contiguous or strided
 subsections, and arrays of booleans to select elements at their `true` indices.
 
-If `X` is an array, it must have the same number of elements as the product of the lengths of
-the indices: `prod(length(I_1), length(I_2), ..., length(I_n))`. The value in location `I_1[i_1], I_2[i_2], ..., I_n[i_n]`
-of `A` is overwritten with the value `X[i_1, i_2, ..., i_n]`. If `X` is a scalar, use the
-element-wise assignment operator `.=` to write the value to all referenced locations of `A`:
+If all indices `Iₖ` are integers, then the value in location `I₁, I₂, ..., Iₙ` of `A` is
+overwritten with the value of `X`, [converting](@ref Base.convert) to the
```

Comment:
This does not work ATM (https://github.com/JuliaDocs/Documenter.jl/issues/781) so you can write this as
```
[`convert`](@ref convert)ing
```